mlflow/gateway/providers/bedrock.py

```diff
@@ -17,6 +17,13 @@
 
 
 class AmazonBedrockAnthropicAdapter(AnthropicAdapter):
+    @classmethod
+    def chat_to_model(cls, payload, config):
```

Comment:
Not related to this PR but I feel like the methods names are quite confusing... It says `chat_to_model` but actually is converting chat request payload to a payload model accepts. And we have no docstrings on the base ProviderAdapter as well :(

Reply:
This is true and sadly some of community-based providers implementation is not as clean as others. We can spend some time cleaning up these after releasing the fetaure.